src/ThemeAssistant.php

```diff
@@ -819,7 +819,7 @@ function executeCssFunction($function, $params)
 		if ($function === 'adjust-hue') {
 
 			$color = substr($params[0], 1);
-			$degrees = $params[1] / 360;
+			$degrees = str_replace('%', '', $params[1]) / 360;
```

Comment:
Just added this line because it was similar to the weights but there could be something like `°` instead of `%`, couldn't it?